lib/backend/k8s/conversion.go

```diff
@@ -361,3 +362,29 @@ func (c converter) k8sPortToCalico(port extensions.NetworkPolicyPort) []numorstr
 	// No ports - return empty list.
 	return []numorstring.Port{}
 }
+
+func (c converter) k8sNodeToCalico(node *kapiv1.Node) *model.KVPair {
+	// Get the internal IP of the node, should be 1-1 with calico node IP
+	nodeIP := ""
+	for _, address := range node.Status.Addresses {
+		if address.Type == kapiv1.NodeInternalIP {
+			nodeIP = address.Address
+			break
+		}
+	}
```

Comment:
What happens if there is no NodeInternalIP specified?

We can probably do one of a few things.

Maybe, if no NodeInternalIP, then:
- use a NodeExternalIP, if none exists then
- use any other configured IP, if none exists then
- treat as "node does not exist"

I wonder if we should be doing anything in the case there are multiple NodeInternalIP addresses specified.  Not sure what we could do though.